src/NHSD.BuyingCatalogue.Solutions.Application/Domain/Solution.cs

```diff
@@ -112,6 +121,10 @@ internal class Solution
             Hosting = string.IsNullOrWhiteSpace(solutionResult.Hosting)
                 ? new Hosting()
                 : JsonConvert.DeserializeObject<Hosting>(solutionResult.Hosting);
+
+            Supplier = string.IsNullOrWhiteSpace(solutionResult.Supplier)
```

Comment:
Please could we go through this change? I'm not sure why we are using Json here.